snuba/subscriptions/scheduler.py

```diff
@@ -321,6 +329,36 @@ def __reset_builder(self) -> None:
             # We are transitioning between jittered and immediate mode. We must use the delegate builder.
             self.__builder = self.__delegate_builder
 
+    def __get_filtered_subscriptions(self) -> List[Subscription]:
```

Comment:
I would recommend that you pass the original list of subscriptions to this method rather than relying on `self.__subscriptions` and rename the method to `__filter_subscriptions`. Looking at the usage of `self.__subscriptions` I don't think there is value in having the private instance variable. You could probably get rid of it completely.

Reply:
Sure, this makes sense. Updated.